6index.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+// task 1:
+
+function addThemAll (...arr) {
+
+    const sum = arr.reduce((accum, item) => {
```

Comment:
змінна sum лишня, можна обійтись без її створення
```suggestion
    return arr.reduce((accum, item) => {
```